lib/__tests__/asset.js

```diff
@@ -1,13 +1,41 @@
-const Asset = require("../asset");
+const Bundler = require('parcel-bundler');
+const inlineSVGPlugin = require('../index');
+const path = require('path');
+const assertBundle = require('parcel-assert-bundle-tree');
+const fs = require('fs');
 
-describe("Asset", () => {
-  it("should ouput the optimized svg code...", (done) => {
-    const asset = new Asset("./lib/__tests__/llama.svg", {}, { rootDir: __dirname });
+const readFile = (location) => new Promise((resolve, reject) => fs.readFile(location, (err, data) => err ? reject(err) : resolve(data)));
 
-    asset.load().then((pluginOutput)=>{
-      expect(pluginOutput).toEqual(`module.exports = '<svg viewBox="0 0 98 115"><text>llaméaa</text></svg>';`)
-      done()
-    })
+describe("Asset", () => {
+  it("should ouput the optimized svg code...", async () => {
+    // Init bundler
+    const bundler = new Bundler(path.join(__dirname, './llama.svg'), {
+      outDir: path.join(__dirname, 'dist'),
+      watch: false,
+      cache: false,
+      hmr: false,
+      logLevel: 0
+    });
+    // Register plugin
+    inlineSVGPlugin(bundler);
 
-  });
+    // Bundle everything
+    const bundle = await bundler.bundle();
+    assertBundle(bundle, {
+      type: 'js',
+      name: 'llama.js',
+      childBundler: [
+        {
+          type: 'map'
+        }
+      ]
+    });
+    
+    // Check asset content
+    expect({
+      bundleContent: (await readFile(bundle.name)).toString()
+    }).toEqual({
+      bundleContent: expect.stringContaining('<svg viewBox="0 0 98 115"><text>llaméaa</text></svg>')
+    });
+  }, 25000);
```

Comment:
You could also change this to a global timeout as every test will have to set this using: jest.setTimeout(25000);